lib/ctblsymm.gi

```diff
@@ -1241,14 +1241,13 @@ InstallMethod( Irr,
 
     # Compute the irreducibles.
     irr:= List( gentbl.matrix( deg ), i -> Character( cG, i{ perm } ) );
-    MakeImmutable( irr );
-    SetIrr( cG, irr );
+    SetIrr( cG, MakeImmutable( irr ) );
```

Comment:
As far as I know, `Irr` is a regular attribute (not a mutable attribute). Therefore any value assigned to this attribute is automatically made immutable, and your code should be equivalent to
```suggestion
    SetIrr( cG, irr );
```